ArrayReverse.java

```diff
@@ -0,0 +1,18 @@
+public class ArrayReverse {
+  public static void main (String[] args) {
+    
+    int[] numbers = {2, 4, 6, 8};
+    int[] newNumbers;
+    newNumbers = new int[4];
+    for (int i =0; i < numbers.length; i++) {
+      int newArray = numbers.length-i-1;
```

Comment:
Minor stuff, but you probably won't want to call an integer newArray. It can lead to misunderstandings if working with a team / code review.